public/indexedit.html

```diff
@@ -0,0 +1,88 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+    <meta charset="utf-8">
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/2.1.4/jquery.min.js"></script>
+    <link rel="stylesheet" type="text/css" href="css.css">
+    <!-- Latest compiled and minified CSS -->
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css" integrity="sha384-BVYiiSIFeK1dGmJRAkycuHAHRg32OmUcww7on3RYdg4Va+PmSTsz/K68vbdEjh4u" crossorigin="anonymous">
+
+    <!-- Optional theme -->
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap-theme.min.css" integrity="sha384-rHyoN1iRsVXV4nD0JutlnGaslCJuC7uwjduW9SVrLvRYooPp2bWYgmgJQIXwl/Sp" crossorigin="anonymous">
+
+    <!-- Latest compiled and minified JavaScript -->
+    <script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/js/bootstrap.min.js" integrity="sha384-Tc5IQib027qvyjSMfHjOMaLkfuWVxZxUPnCJA7l2mCWNIpG9mGCD8wGNIcPD7Txa" crossorigin="anonymous"></script>
+    <title></title>
+</head>
+  <body>
+    <form id="editForm">
+      <h2>Edit a Movie</h2> Title:
+
+      <br>
+      <input type="text" name="title">
+      <br> Director:
+      <br>
+      <input type="text" name="director">
+      <br> Year:
+      <br>
+      <input type="text" name="year">
+      <br> Rating:
+      <br>
+      <input type="text" name="myRating">
+      <br> PosterUrl:
+      <br>
+      <input type="text" name="posterUrl">
+      <br>
+
+      <br>
+      <input type="submit" id="editMovieButton" value="Submit">
+  </form>
+
+  <script src="movies.js"></script>
+
+  <script>
+  $( document ).ready(function() {
+  function getUrlVars(){
+    var vars = [], hash;
+    var hashes = window.location.href.slice(window.location.href.indexOf('?') + 1).split('&');
+    for(var i = 0; i < hashes.length; i++)
+    {
+        hash = hashes[i].split('=');
+        vars.push(hash[0]);
+        vars[hash[0]] = hash[1];
+    }
+    return vars;
+}
+
+var queryParams = getUrlVars();
+console.log(queryParams, 'queryParams');
+
+var id = Object.values(queryParams)[1];
+console.log(id,'id');
+$.get('/movies/?id='+id, function(res){
+var movie = res.find(function(testMovie){
```

Comment:
Everything inside of `function(res){` needs to be indented to let readers know they are inside the function.